src/npcmove.cpp

```diff
@@ -3373,12 +3373,12 @@ bool npc::wield_better_weapon()
         }
 
         if( val > best_value ) {
-            best = const_cast<item *>( &it );
+            best = const_cast<item &>( it );
```

Comment:
It would be more efficient to keep "best" as a pointer in order to avoid making a copy of the item here (i.e. `best = const_cast<item *>( &it )` )